Content.Server/Ghost/Roles/Events/GhostRoleSpawnerUsedEvent.cs

```diff
@@ -0,0 +1,22 @@
+namespace Content.Server.Ghost.Roles.Events
+{
+    public sealed class GhostRoleSpawnerUsedEvent : EntityEventArgs
+    {
+        /// <summary>
+        /// The entity that spawned this.
+        /// </summary>
+        public EntityUid Spawner = new();
+
+        /// <summary>
+        /// The entity spawned.
+        /// </summary>
+        public EntityUid Spawned = new();
+
+        public GhostRoleSpawnerUsedEvent(EntityUid spawner, EntityUid spawned)
+        {
+            Spawner = spawner;
+
+            Spawned = spawned;
```

Comment:
```suggestion
            Spawner = spawner;
            Spawned = spawned;
```

Reply:
should it even include fields at all? the spawner is presumably being deleted most of the time (so this should at least be nullable), and its being raised on the spawned entity, so

Reply:
It lets us get information about the spawner in the frame before it's deleted. It's directly used in this PR

Reply:
alright, thats fine then